spread/lxd.go

```diff
@@ -156,6 +221,17 @@ func isDebArch(s string) bool {
 	return false
 }
 
+func sshInstallCommand(distro Distro) []string {
+	if distro == Ubuntu || distro == Debian {
+		return []string{"apt", "install", "openssh-server"}
```

Comment:
This seems to be fixing Ubuntu and Debian, not Fedora, as the PR title suggests, but having spoken to people behind linux-containers.org, ssh servers are not installed in those images on purpose, for security reasons (default ssh server settings are often too open).